index.js

```diff
@@ -1,8 +1,8 @@
 const isWeekend = date => {
   const dateString = date;
   const weekDay = dateString.getDay();
-  const hour = ((dateString.getHours() + 11) % 12 + 1);
-  return ((weekDay === 5 && hour >= 5) || weekDay === 6 || weekDay === 0);
+  const hour = dateString.getHours() + 1;
```

Comment:
Maybe the `dateString` variable can be removed? Accessing `date` directly seems more straightforward.

Reply:
Not in scope of this PR